### PR TITLE
[#147674123] Extend process for dealing with RDS failure

### PR DESCRIPTION
**Pull request by paroxp**
## What

It's not very explicit that we should contact our users when their RDS
instance experiences failure events from AWS.

We're slightly reforming these sections to reflect that in a better
manner.

## How to review

Read through the docs and see if the changes make sense.
Establish whether it's implying the need of contacting our tenants on occasion.

### CODE REVIEW

File: docs/support/responding_to_alerts.md [hunk 1]
```diff
@@ -59,6 +59,12 @@ resolvable without getting in touch with AWS. In some circumstances you may be
 able to restore an instance using [our point-in-time-restore
 instructions](../guides/Restoring_the_CF_databases.md).
 
+When `failure` monitor triggers, before starting any further investigation and
+work, we would like to notify our users of odd behaviour and let them know
+we're working to solve this issue. Please see the [How to obtain organisation
+managers](#how-to-obtain-organisation-managers) to learn who should be
+contacted on occasion.
```
Comment (bleach) [inline docs/support/responding_to_alerts.md]: I think we can remove the "on occasion". If we were going to keep it, it should probably be "on this occasion".

File: docs/support/responding_to_alerts.md [hunk 2]
```diff
@@ -71,7 +77,11 @@ happens.
 
 Ideally, you should retrieve a GUID of an instance from the tags associated
 with the metrics on the alert. Having that will help you establish the users to
-contact, as they may wish to scale up their instance in response.
+contact, as they may wish to scale up their instance in response. Please see
+the [How to obtain organisation managers](#how-to-obtain-organisation-managers)
+to learn who should be contacted on occasion.
```
Comment (bleach) [inline docs/support/responding_to_alerts.md]: Same comment as above.

File: docs/support/responding_to_alerts.md (hunk 1 above)
Comment (LeePorte) [inline docs/support/responding_to_alerts.md]: I'd suggest changing to `to learn who to contact.`

File: docs/support/responding_to_alerts.md (hunk 2 above)
Comment (LeePorte) [inline docs/support/responding_to_alerts.md]: As above.

File: docs/support/responding_to_alerts.md (hunk 1 above)
Comment (paroxp) [reply docs/support/responding_to_alerts.md]: addressed

Comment (paroxp) [reply docs/support/responding_to_alerts.md]: addressed

**Comment by bleach** [comment]
Great. Looks good now!

**Pull request merged**